### ISSUE TITLE
Testing Zelerius algorithm - Block V7

### ISSUE THREAD

**Issue by nilhcraiv** [zelerius/Zelerius-Network#8]
Zelerius Network is switching from CN variant 2 (Monero V8) to CN variant Zelerius ( this is cryptonight_monero_v8 but with fewer iterations -> 393216 / 0x60000 ) The algorithm change will be configured in the Block Major Version 7.

**Test Network**

We are testing at https://xyztest.zelerius.org

**Block version**

```
- Block Major Version 6: CN Variant Monero V8 - cryptonight_v8
- Block Major Version 7: CN Variant Zelerius - cryptonight_zelerius
```

**Comment by zelerius** [comment]
It seems we have an issue with AMD using xmr-stk
https://github.com/fireice-uk/xmr-stak/issues/2262

SRBMiner works well with Cryptonight Zelerius
https://www.srbminer.com

**Comment by zelerius** [comment]
Included in v1.4.0
